packages/framework-provider-aws/package.json

```diff
@@ -38,7 +38,7 @@
     "@types/aws-lambda": "^8.10.48",
     "@types/faker": "^4.1.12",
     "@types/get-installed-path": "^4.0.0",
-    "aws-sdk": "^2.771.0",
+    "aws-sdk": "2.771.0",
```

Comment:
This is something that we should take into account much more closely from now on: use strict versions, and, as you say, use the same version everywhere.

I think we were using the same version in both packages, so I will change this. Thanks!

Maybe there is a way to parameterize the `package.json` files so that we can use variables for common version numbers...